tests/test_ote_training.py

```diff
@@ -145,10 +146,12 @@ def _convert(p):
 
 class OTETrainingImpl:
     def __init__(self, dataset_params: DatasetParameters, template_file_path: str,
-                 num_training_iters: int):
+                 num_training_iters: int, batch_size: int):
+        logger.info(f'::: DEBUG::: dataset_params={dataset_params},\n template_file_path={template_file_path},\n num_training_iters={num_training_iters},\n batch_size={batch_size}')
```

Comment:
Is it info of debug?

Reply:
Ooops, forgot to remove this debug insertion